LibDmd/Output/ZeDMD/DMDESP32.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.IO.Ports;
+
+namespace DMDESP32
+{
+	public class DMD_ESP32 // Class for locating the COM port of the ESP32 and communicating with it
+	{
+		public string nCOM;
+		public int BaudRate;
+		public byte ByteSize = 8;
+		public bool Opened = false;
+		public byte Parity = 0; // 0-4=no,odd,even,mark,space
+		public string Port;
+		public int ReadTimeout;
+		public byte StopBits = 1; // 0,1,2 = 1, 1.5, 2*/
+		private SerialPort _serialPort;
+		private static readonly int _MAX_SERIAL_WRITE_AT_ONCE = 9400;
+		public static readonly int N_CTRL_CHARS = 6;
+		public static readonly int N_INTERMEDIATE_CTR_CHARS = 4;
+		public static readonly byte[] CtrlCharacters = { (byte)0x5a, (byte)0x65, (byte)0x64, (byte)0x72, (byte)0x75, (byte)0x6d };
+		private bool _Scom_Connect(string port, ref int rx, ref int ry)
+		{
+			try
+			{
+				BaudRate = 921600;
+				Port = port;
+				_serialPort = new SerialPort(port, BaudRate, System.IO.Ports.Parity.None, 8, System.IO.Ports.StopBits.One);
+				_serialPort.ReadTimeout = 100;
+				_serialPort.WriteBufferSize = _MAX_SERIAL_WRITE_AT_ONCE + 100;
+				_serialPort.WriteTimeout = SerialPort.InfiniteTimeout;

```

Comment:
You can do this in C# as well (at your discretion):
```suggestion
				_serialPort = new SerialPort(port, BaudRate, System.IO.Ports.Parity.None, 8, System.IO.Ports.StopBits.One) {
					ReadTimeout = 100,
					WriteBufferSize = _MAX_SERIAL_WRITE_AT_ONCE + 100,
					WriteTimeout = SerialPort.InfiniteTimeout
				};

```